frozen-abi/src/abi_digester.rs

```diff
@@ -550,7 +550,7 @@ mod tests {
         test_field2: i8,
     }
 
-    #[frozen_abi(digest = "Hv597t4PieHYvgiXnwRSpKBRTWqteUS4nHZHY6ZxX69v")]
+    #[frozen_abi(digest = "GMeECsxg37a5qznstWXeeX3d6HXs6j12oB4SKaZZuNJk")]
```

Comment:
@ryoqun - These two hashes changed on me for some reason.  Seems harmless though.  Maybe it's due to the crate rename somenow?

Reply:
yeah, it detects such renames because it need to consider XX::Hash to be different  from YY::Hash.

```patch
$ diff -U3 /tmp/abi8/*Hv597t4PieHYvgiXnwRSpKBRTWqteUS4nHZHY6ZxX69v* /tmp/abi8/*GMeECsxg37a5qznstWXeeX3d6HXs6j12oB4SKaZZuNJk*
--- /tmp/abi8/abi_digester__tests__TestNest_frozen_abi__test_abi_digest_Hv597t4PieHYvgiXnwRSpKBRTWqteUS4nHZHY6ZxX69v    2020-10-20 21:17:01.936993559 +0900
+++ /tmp/abi8/abi_digester__tests__TestNest_frozen_abi__test_abi_digest_GMeECsxg37a5qznstWXeeX3d6HXs6j12oB4SKaZZuNJk    2020-10-20 21:21:09.115486766 +0900
@@ -1,31 +1,31 @@
 struct TestNest (fields = 1)
-    field nested_field: [solana_sdk::abi_digester::tests::TestStruct; 5]
+    field nested_field: [solana_frozen_abi::abi_digester::tests::TestStruct; 5]
         tuple (elements = 5)
-            element solana_sdk::abi_digester::tests::TestStruct
+            element solana_frozen_abi::abi_digester::tests::TestStruct
                 struct TestStruct (fields = 2)
                     field test_field: i8
                         primitive i8
                     field test_field2: i8
                         primitive i8
-            element solana_sdk::abi_digester::tests::TestStruct
+            element solana_frozen_abi::abi_digester::tests::TestStruct
                 struct TestStruct (fields = 2)
                     field test_field: i8
                         primitive i8
                     field test_field2: i8
                         primitive i8
-            element solana_sdk::abi_digester::tests::TestStruct
+            element solana_frozen_abi::abi_digester::tests::TestStruct
                 struct TestStruct (fields = 2)
                     field test_field: i8
                         primitive i8
                     field test_field2: i8
                         primitive i8
-            element solana_sdk::abi_digester::tests::TestStruct
+            element solana_frozen_abi::abi_digester::tests::TestStruct
                 struct TestStruct (fields = 2)
                     field test_field: i8
                         primitive i8
                     field test_field2: i8
                         primitive i8
-            element solana_sdk::abi_digester::tests::TestStruct
+            element solana_frozen_abi::abi_digester::tests::TestStruct
                 struct TestStruct (fields = 2)
                     field test_field: i8
                         primitive i8

```

```
$ sha256sum /tmp/abi8/abi_digester__tests__TestNest_frozen_abi__test_abi_digest_Hv597t4PieHYvgiXnwRSpKBRTWqteUS4nHZHY6ZxX69v | xxd -r -p - | base58 && echo
Hv597t4PieHYvgiXnwRSpKBRTWqteUS4nHZHY6ZxX69v

$ sha256sum /tmp/abi8/abi_digester__tests__TestNest_frozen_abi__test_abi_digest_GMeECsxg37a5qznstWXeeX3d6HXs6j12oB4SKaZZuNJk | xxd -r -p - | base58 && echo
GMeECsxg37a5qznstWXeeX3d6HXs6j12oB4SKaZZuNJk

```